### ISSUE TITLE
[BUG] False Positive with sprinkles() use inside recipies

### ISSUE THREAD

**Issue by skinread** [antebudimir/eslint-plugin-vanilla-extract#7]
name: Bug Report
about: Report an issue with the vanilla-extract CSS ESLint plugin
title: '[BUG]: False Positive with sprinkles() Function Calls'
labels: bug, needs-triage
assignees: ''

## Contribution Checklist

- [x] I have read documentation and understand the feature
- [x] I have searched for similar issues before creating this one
- [x] I have tested this with the latest plugin version
- [x] I have included all information needed to reproduce the issue

### Bug Description

The `vanilla-extract/no-empty-style-blocks` ESLint rule incorrectly flags sprinkles() function calls within recipe variants as empty style blocks, even though they contain valid style definitions. The rule appears to only recognise style() calls and not certain style-generating functions like sprinkles().

### Reproduction Steps

1. Create a vanilla-extract recipe with variants that use sprinkles() function calls
2. Include a variant object with an empty key (e.g., wrappingDirection: { reverse: ... } without a default key)
3. Run ESLint with the vanilla-extract/no-empty-style-blocks rule enabled

### Expected Behavior

The ESLint rule should recognize that sprinkles() function calls are valid style-generating functions and not flag them as empty style blocks, regardless of whether variant keys appear empty in the recipe structure.

Actual Behavior

The rule incorrectly reports lint errors on valid sprinkles() function calls within recipe variants, treating them as empty style blocks when they clearly contain style properties.

## Code Example

```js
import { recipe } from '@vanilla-extract/recipes';
import { sprinkles } from '@vanilla-extract/sprinkles';

export const columnsStyle = recipe({
    base: sprinkles({ display: 'flex', flexDirection: 'row' }),
    variants: {
        wrappingDirection: {
            // False positive: rule flags this as empty despite being a valid sprinkles() call
            reverse: sprinkles({ flexWrap: 'wrap-reverse' }),
        },
    },
});
```

## Environment Information

- Node Version: 22.20.0
- Package Versions:
  - ESLint: 9.38.0
  - @vanilla-extract/css: 1.17.4
  - TypeScript: 5.9.3
  - @antebudimir/eslint-plugin-vanilla-extract: 1.15.0

- [ ] Blocks critical functionality
- [ ] Produces false positives
- [ ] Misses valid errors
- [ ] Performance issue
- [ ] Other (please specify)

**Comment by antebudimir** [comment]
@skinread,

Fixed it, Sean. Please check it out when you find time and confirm that all's good on your end.
